test/envtest/reconciler_setupwithmanager_test.go

```diff
@@ -71,10 +93,24 @@ var konnectGatewayControlPlaneTestCases = []konnectEntityReconcilerTestCase{
 							Name: "auth",
 						},
 					},
+					CreateControlPlaneRequest: sdkkonnectcomp.CreateControlPlaneRequest{
+						Name:        "cp-1",
+						Description: lo.ToPtr("test control plane 1"),
+					},
 				},
 			}
 			require.NoError(t, cl.Create(ctx, cp))
 		},
+		mockSDKSettings: func(t *testing.T, m *ops.MockSDKWrapper, ns *corev1.Namespace) {
```

Comment:
`testing.T` seems unused now. I wonder if there's a chance this will be needed at some point 🤔